_users.md

```diff
@@ -87,7 +87,8 @@ curl "https://api.uphold.com/v0/me" \
 
 ### Response
 
-Returns the details associated the current user.
+Returns the data associated the current user.
```

Comment:
Missing `with`.

Reply:
Oops, missed that one. I'll edit in a bit.

Reply:
Fixed.